src/components/Book.js

```diff
@@ -1,13 +1,14 @@
 /* eslint-disable */
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { bookRemoved, BOOK_REMOVED } from '../redux/reducers/books';
+import { removeBook } from '../redux/reducers/books';
 import Swal from 'sweetalert2';
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
 
-  const { id, title, author, chapter, completed } = book;
+  const { item_id, title, author, chapter = '', completed = '0%' } = book;
+  console.log('item id  :', item_id, 'title : ', title, 'author : ', author);
 
   const handleDelete = (id) => {
     const swalWithBootstrapButtons = Swal.mixin({
```

Comment:
- [ ] Great job adding a confirmation alert for delete action.

![image](https://user-images.githubusercontent.com/68796232/200918168-6fd816aa-492b-4978-9ef4-07703934e1bc.png)